server/cf.conf

```diff
@@ -36,6 +36,8 @@ procs = cf
 
 [cf]
 # cf-store application
+# a space-separated list of infotags to set as CF Properties
+#infotags = archive foo bar blah
```

Comment:
Inconsistent formatting: on this line `infotags` has no space before it, but on the other lines that should be uncommented there is a space. 🔧

Reply:
This convention is intended to distinguish the line with the config file key from the explanatory comment.

Reply:
I thought that might be the case, but it doesn't match the lines below. Either way seems good to me.